src/css/abstracts/_constants.scss

```diff
@@ -59,5 +59,14 @@ $grid-names: (
     lg: lg
 );
 
+// selection of column utility classes for each breakpoint
+$grid-classes: (

```

Comment:
@lewisbilsland this defines the classes that are generated by the grid function, so it doesn't generate every className xm/sm/md/lg 1-12. I think that's a good thing, what do you reckon? And are these combinations of size and column a reasonable default?